views/api/charts/chart/index.js

```diff
@@ -3,9 +3,9 @@ import { apiRoutes } from '../../../routes/routes'
 
 const chart = {
   create: async (chartAttributes) =>
-    http.post(apiRoutes.chart.index, chartAttributes),
+    http.post(apiRoutes.chart.all, chartAttributes),
   destroy: async (chart_id) => http.delete(apiRoutes.chart.show(chart_id)),
-  index: async () => http.get(apiRoutes.chart.index),
+  all: async (queryParams) => http.get(apiRoutes.chart.all, queryParams),
```

Comment:
this should still be the index route.

Reply:
Okay, we've always had it as index, there was some feedback about converting it to all. 
https://github.com/AMP-SCZ/dpdash/pull/557#discussion_r1436475609

Reply:
that comment is related to the method name, I believe. The route should be index.

Reply:
👍

Reply:
and that makes more sense.
[216b66e](https://github.com/AMP-SCZ/dpdash/pull/560/commits/216b66e967a28fae64661657855e58bd0984fef3)